Failure to build for 32 bit

```
error[E0308]: mismatched types
  --> crates/auparse/sys/src/event.rs:28:18
   |
27 |     pub fn ts(&self) -> i64 {
   |                         --- expected `i64` because of return type
28 |         unsafe { auparse_get_time(self.au.as_ptr()) }
   |                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `i64`, found `i32`
   |
help: you can convert an `i32` to an `i64`
   |
28 |         unsafe { auparse_get_time(self.au.as_ptr()).into() }
   |                                                    +++++++
For more information about this error, try `rustc --explain E0308`.
```

--- a/crates/auparse/sys/src/event.rs
+++ b/crates/auparse/sys/src/event.rs
@@ -25,7 +25,10 @@ impl Event {
     }
 
     pub fn ts(&self) -> i64 {
-        unsafe { auparse_get_time(self.au.as_ptr()) }
+        #[allow(clippy::useless_conversion)]
+        unsafe {
+            auparse_get_time(self.au.as_ptr()).into()
+        }
     }
     pub fn int(&self, name: &str) -> Result<i32, Error> {
         unsafe { audit_get_int(self.au.as_ptr(), name) }

--- a/fapolicy-analyzer.spec
+++ b/fapolicy-analyzer.spec
@@ -58,10 +58,6 @@ Requires:      gnome-icon-theme
 Requires:      webkit2gtk3
 Requires:      mesa-dri-drivers
 
-# rust-ring-devel does not support s390x and ppc64le:
-# https://bugzilla.redhat.com/show_bug.cgi?id=1869980
-ExcludeArch:   s390x %{power64}
-
 %global module          fapolicy_analyzer
 # pep440 versions handle dev and rc differently, so we call them out explicitly here
 %global module_version  %{lua: v = string.gsub(rpm.expand("%{?version}"), "~dev", ".dev"); \

--- a/scripts/srpm/fapolicy-analyzer.el9.spec
+++ b/scripts/srpm/fapolicy-analyzer.el9.spec
@@ -131,10 +131,6 @@ Requires:      gnome-icon-theme
 Requires:      webkit2gtk3
 Requires:      mesa-dri-drivers
 
-# rust-ring-devel does not support s390x and ppc64le:
-# https://bugzilla.redhat.com/show_bug.cgi?id=1869980
-ExcludeArch:   s390x %{power64}
-
 %global module          fapolicy_analyzer
 
 %global venv_dir        %{_builddir}/vendor-py

